ci/checks/enforce-correct-shebangs.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -e -o pipefail -u
+
+if grep -r -e '^#!\/usr\/bin\/env python$' -e '^#!\/usr\/bin\/env python2$' ../../git_machete; then
```

Comment:
No need to escape `/` in regexes

Reply:
Also, should it be `../../git_machete` or just `git_machete`? 🤔  since AFAICS this script is executed from the top level folder in the CI